frontend/public/components/image-stream.tsx

```diff
@@ -111,7 +111,7 @@ const ImageStreamTagsRow: React.SFC<ImageStreamTagsRowProps> = ({
         {from && referencesTag && (
           <ResourceLink
             kind={ImageStreamTagsReference}
-            name={getImageStreamTagName(imageStream.metadata.name, from.name)}
+            name={from.name}
```

Comment:
The referenceTag check on the top will make sure, if the tag is an alias of another ImageStreamTag, so in this case from.name will always contain the referenced IST name (echoenv:test-one), right?